_ingest-pipelines/processors/ml-inference.md

```diff
@@ -0,0 +1,158 @@
+| `max_prediction_tasks` | Int | Optional | The setting of the maximum number of prediction tasks can run when ingesting a document, default value is 10.                                                                                                                                                                                                                                                                                                     |
+| `description` | String | Optional | A brief description of the processor.                                                                                                                                                                                                                                                                                                                                                                             |
+| `tag` | String | Optional | An identifier tag for the processor. Useful for debugging to distinguish between processors of the same type.                                                                                                                                                                                                                                                                                                     |
+| `ignore_missing` | Boolean | Optional | If true and any of the input fields defined in input_map or output_map are missing, those missing fields are quietly ignored. Otherwise, a missing field causes a failure. Default is false.                                                                                                                                                                                                                      |
+| `ignore_failure` | Boolean | Optional | If true and any exceptions occur, they are quietly ignored, and the ingestion continues. Otherwise, an exception can occur and abort the ingestion. Default is false.                                                                                                                                                                                                                                             |
+
+## Using the processor
+
+Follow these steps to use the processor in a pipeline. You must provide a model ID when creating the processor. For more information, see [Using custom models within OpenSearch]({{site.url}}{{site.baseurl}}/ml-commons-plugin/using-ml-models/).
+
+**Step 1: Create a pipeline.**
+
+The following example request creates an ingest pipeline where the model requires an `input` field and produces a `data` field in the prediction task and the document has a `passage_text` field, which  will be converted into text embeddings and the embeddings will be stored in `passage_embedding` field. 
```

Comment:
Suggest create a model with one of our blueprints https://github.com/opensearch-project/ml-commons/tree/2.x/docs/remote_inference_blueprints, that will make it easier for user to create connector and model and try the ml_inference example below

Reply:
will link with the tutorial and starting from creating connector, creating model, deploying model, creating index and upload documents.

Reply:
@mingshl Have you added the tutorial? Once it's added, please tag me for the doc review. I'll also move it forward into editorial review.

Reply:
not yet. will create seperate PRs for the tutorials. I think this Pr is ready to merge/ Thanks @vagimeli